pkg/processor/processor.go

```diff
@@ -178,6 +173,12 @@ func (runner *ReportRunner) Run(reportFn func(report *ReportToExecute) (map[stri
 	for _, report := range runner.Reports {
 		var err error
 
+		caseNumber, err := common.GetCaseNumberByFilename(report.File.Path)
+		if err != nil || caseNumber == "" {
```

Comment:
I think its neater len(caseNumber) == 0

Reply:
agreed but im going to stick with the existing convention for this one